packages/schematics/angular/migrations/update-14/update-tsconfig-target.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { JSONFile } from '../../utility/json-file';
+import { getWorkspace } from '../../utility/workspace';
+import { Builders } from '../../utility/workspace-models';
+
+const SUPPORTED_BUILDERS = [Builders.Karma, Builders.NgPackagr, Builders.Browser];
+
+/** Migration to update tsconfig compilation target option to es2020. */
+export default function (): Rule {
+  return async (host) => {
+    // Workspace level tsconfig
+    updateTargetToES2020(host, 'tsconfig.json');
```

Comment:
Question: Does Angular require that the main tsconfig is named `tsconfig.json`? Could users rename this and would it fail the migration if so? I'm thinking we just need to check if the file exists and only migrate it if so.

Reply:
Technically they can rename it, but I don't think that anyone does because;

1) This is the tsconfig that is used by the IDE - without it the TS language service will not work correctly. 
2) This tsconfig is expected to be there by other schematics that we have, such as `ng g app`, `ng g lib` etc...

Reply:
Since there would be other problems from renaming the base `tsconfig.json` anyways, I'm not too concerned about this one.